client/app/hooks/stickyHeader.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+export const stickyHeader = () => {
```

Comment:
If we implement this sticky header completely in React, then we don't need to be making these DOM manipulations to toggle the sticky header.

Reply:
I tried just doing that, and didn't manage to make it work. The reason, I think, is that React on Rails inserts each react component as an independent div node, preventing its CSS style sheet any access to its ancestor elements in the DOM. Moreover, the intersection observability api which is used to change the color when scrolling down, also needs access to an ancestor element to the header.

I am open to any suggestion that would allow only modifying the react header component code, but I'm afraid that would require a different approach.